### DISCUSSION TITLE
I can run "A1111" just fine, but this fork - not so much

### DISCUSSION THREAD

**Discussion by space-hamster0442** [vladmandic/sdnext#493] [Q&A] [answered]
Hello.
I can run "A1111" just fine, but this fork - not so much...

windows 11
16gb ram, 2060super, 6gb vram (laptop version)
Python version: 3.10.11, Git: 2.40.0, Cuda: 12.1.1_531.14

I'm getting "CUDA Out of memory" and can't generate a single image. My settings for running "A1111" were "set COMMANDLINE_ARGS= --medvram --precision full --no-half --xformers", "set PYTORCH_CUDA_ALLOC_CONF=garbage_collection_threshold:0.6,max_split_size_mb:512". 

Maybe you can help me with specific settings to run this fork, I really want to try it, I'm getting so much errors on "A1111" with the latest update(s). 

edit: sorry, if this question was asked before, I'm new here

**Comment by space-hamster0442** [comment]
Sorry, did a "clean" reinstall, now images are generating... But I still don't understand how to add command line params here...

**Reply by gsgoldma** [reply, marked as answer]
dont think you need to. i have 6 gigs and can make an 512 bs 512 image in 3-4 seconds using only 30-50% of my vram

**Comment by TingTingin** [comment]
the command line params are moved to the ui so you can edit them from the ui directly

**Comment by space-hamster0442** [comment]
Thank you for your help, looks like I have a new problem...
When I try to use "Interrogate CLIP" in img2img I get this:
![smthng](https://user-images.githubusercontent.com/131806797/234824510-073c6a2e-c7e5-47bb-82ea-02ed507a3d87.png)
...in txt2img too, when I try to use hires.fix
edit: small typos

**Comment by vladmandic** [comment]
@space-hamster0442 that has nothing to do with original issue and it seems like download of model is not working

**Reply by space-hamster0442** [reply]
Sorry, just thought about it but decided to write about it here anyway, you can move it... 
Thought that it still can be linked with the original topic.
